components/function-controller/README.md

```diff
@@ -25,7 +25,7 @@ The Function Controller uses these environment variables:
 | **APP_LEADER_ELECTION_ID**                        | Name of the ConfigMap that specifies the main instance of the Function Controller that manages the traffic among all instances                                                                                                                                                                  | `serverless-controller-leader-election-helper`                                                                                                             |
 | **APP_CONFIG_WATCHER_ENABLE_CONTROLLERS**         | Field that enables Config Watcher controllers that propagate ConfigMaps and Secrets to all Namespaces                                                                                                                                                                                        | `true`                                                                                                                                                     |
 | **APP_CONFIG_WATCHER_BASE_NAMESPACE**             | Name of the Namespace with the serverless configuration (such as runtime, Secret and service account for the Docker Registry) propagated to other Namespaces                                                                                                                                                                    | `kyma-system`                                                                                                                                              |
-| **APP_CONFIG_WATCHER_EXCLUDED_NAMESPACES**        | List of Namespaces to which serverless configuration is not propagated                                                                                                                                                | `istio-system;knative-eventing;knative-serving;kube-node-lease;kube-public;kube-system;kyma-installer;kyma-integration;kyma-system;tekton-pipelines;natss` |
```

Comment:
Could you also update it here: https://github.com/kyma-project/kyma/blob/master/components/function-controller/internal/configwatcher/configwatcher.go#L21 ?